### DISCUSSION TITLE
Changing pitch of hitsounds for early/late indicator

### DISCUSSION THREAD

**Discussion by meltq** [ppy/osu#25378] [Ideas]
If you hit a note early, the hit sound gets slightly higher pitched, if you hit it late, hit sound gets lower pitch. Not sure how this would play irl but it feels like a convenient way to judge how you're playing as most players like to have high hit sound volume and follow that sound. You can figure out of you're hitting early or late basically instantaneously this way without having to move your attention away from the hit circles. Just a slight pitch change cus it would probably be too jarring otherwise.

**Comment by peppy** [comment]
Could be something interesting to play with. cc @nekodex
